tests/functional/test_endpoints_v2.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2012-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import json
+import os
+
+import pytest
+
+from botocore.endpoints_v2 import (
+    EndpointProvider,
+    EndpointResolutionError,
+    RuleSetEndpoint,
+)
+from botocore.loaders import Loader
+
+
+@pytest.fixture
+def partitions():
+    loader = Loader()
+    return loader.load_data('partitions')
+
+
+def ruleset_testcases(case_type):
+    loader = Loader()
```

Comment:
Loader should be able to be module scope either as a fixture or a constant.

Reply:
yup this was fixed right after you commented this!